.github/workflows/linux.yml

```diff
@@ -0,0 +1,101 @@
+name: Linux CI
+
+on: [push, pull_request]
```

Comment:
Won't this produce duplicate builds?  I.e. one for the push and another for the PR?

Reply:
No.

Reply:
Well it *does* produce duplicates in some sense, when core contributors push a branch to the main repo and then create a PR from that. It's possible to tell it to test only when the push is to master, but then you lose the ability to *just* push a branch *just* for the purpose of running CI on it.

So this is fine. It's been this way in win.yml since the start, anyway.